### DISCUSSION TITLE
Localization to different languages

### DISCUSSION THREAD

**Discussion by demirdev** [pocketbase/pocketbase#1214] [Q&A] [answered]
How we localize 
- email templates 
- password reset form labels
- email verification success message
and another text that users can see. 🙂

**Comment by ganigeorgiev** [comment, marked as answer]
> email templates

Currently there is no integrated support for email templates localization, but there are workarounds. You can check https://github.com/pocketbase/pocketbase/issues/593.

> password reset form labels
> email verification success message

You can define your own custom password reset, verification, etc. pages by changing the "Action url" field in the related mail templates.

All api error responses (eg. for fields validation) comes with `code` key that you can use to retrieve your own localized version, eg.:

```json
{
  "code": 400,
  "message": "Failed to update record.",
  "data": {
    "title": {
      "code": "validation_required",
      "message": "Missing required value."
    }
  }
}
```
